spec/spec_helper.rb

```diff
@@ -29,4 +29,7 @@
   # automatically. This will be the default behavior in future versions of
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
+
+  config.include Capybara::DSL
```

Comment:
The DSL is included by default for anything in `spec/features` but not `spec/integration`.

I am not entirely sure how to could move this to `spec/support/capybara.rb`.